compute_tools/src/checker.rs

```diff
@@ -0,0 +1,48 @@
+use std::sync::{Arc, RwLock};
+
+use anyhow::{anyhow, Result};
+use log::error;
+use tokio_postgres::NoTls;
+
+use crate::zenith::ComputeState;
+
+pub async fn check_write_ability(state: &Arc<RwLock<ComputeState>>) -> Result<()> {
+    let connstr = state.read().unwrap().connstr.clone();
+    let (client, connection) = tokio_postgres::connect(&connstr, NoTls).await?;
+    if client.is_closed() {
+        return Err(anyhow!("connection to postgres closed"));
+    }
+    tokio::spawn(async move {
+        if let Err(e) = connection.await {
+            error!("connection error: {}", e);
+        }
+    });
+
+    let result = client
+        .simple_query(
+            "
+        CREATE TABLE IF NOT EXISTS health_check (
+             id serial primary key,
+            updated_at timestamptz default now()
+        );
```

Comment:
I thought about adding this (both table create and test row insert) as a step into compute startup routine, so we only did it once on startup. I don't think that `CREATE TABLE IF NOT EXISTS` is expensive, but 1) it just looks cleaner and 2) here we can do an update by id, so if for some reason we will loose table, test row, or btree got damaged we will notice that. Otherwise, we may tolerate a lot of errors.

Not 100% sure actually that what I propose is better, just some thoughts

Reply:
@ololobus What is the best place to insert this creation?